lte/protos/pipelined.proto

```diff
@@ -325,10 +325,11 @@ message PDI {
   uint32 src_interface = 1;
   uint32 local_f_teid = 2;
   string net_instance = 3;
-  string ue_ip_adr = 4;
+  string ue_ipv4 = 4;
   bytes tr_ep_id = 5;                //8.2.92
   SdfFilters sdf_filters = 6;
   string app_id = 7;
+  string ue_ipv6 = 8;
```

Comment:
any reason for not using IPAddress, we could  avoid conversion of IP address (convert_ip_str_to_ip_proto()) if IPAddress is used in API.

Reply:
@pshelar , Yes, because for AMF-SMF grpc is using string only and same grpc message used for 4g as well.
So, either Sessiond can be converted into string to IP or pipelined.

If Sessiond will do, then again ```ActivateFlowsRequest``` need to change for it. This message is used for 4g and 5g and it could be impact on 4g as well.